### ISSUE TITLE
suggested model for fineTuning paraphrase_mining model for binary classification task?

### ISSUE THREAD

**Issue by khieu** [UKPLab/sentence-transformers#1156]
I saw in `sentence_transformers.util` have the `paraphrase_mining` tool that can rank a set of sentences by similarity metric
I would like to fineTune the a paraphraser network into a binary classification network that returns whether 2 sentences are a paraphrase of each other or not. Can anyone give a suggestion on which model I should fine-tune to the task? For context, the `paraphrase_mining` tool is shown in this doc [https://www.sbert.net/examples/applications/paraphrase-mining/README.html] which uses this model `paraphrase-MiniLM-L6-v2`.  
Also, any suggested training script for classification task is greatly appreciated!
Thanks so much for any help and suggestion.

**Comment by nreimers** [comment]
Classification is simple:
Compute the cosine similarity for the two embeddings
Is the cosine similarity above a certain threshold => you consider them as paraphrases.

The threshold depends on your needs: Do you need more precision or more recall.

No need to add a binary classification layer or something like this on top
